.github/workflows/check-lambda-js.yml

```diff
@@ -10,15 +10,16 @@ on:
       - main
       - development
       - ia-web-development
+      - ia-development
     paths:
-      - backend/compact-connect/lambdas/cloudfront-csp/**
+      - backend/compact-connect/lambdas/nodejs/**
 
   # Allows you to run this workflow manually from the Actions tab
   workflow_dispatch:
 
 # A workflow run is made up of one or more jobs that can run sequentially or in parallel
 jobs:
-  CheckLambdaJS:
+  CheckLambdas:
     runs-on: ubuntu-latest
 
```

Comment:
# Accessibility: Keyboard Navigation

When the screen is small enough to collapse the menu into an expandable menu, I can no longer navigate the menu with my keyboard (the menu is skipped when I tab through items). I realize this is technically mobile view, but the menu seems to transition at a width that I might still use on a laptop.

Reply:
I think this might be the longstanding Safari issue of not allowing tab nav to link / anchor elements by default. Unfortunately this is only something that can be changed in the client-side browser settings. There is technically a app code hack-around, but it negatively impacts screen reader browsers.

https://www.a11yproject.com/posts/macos-browser-keyboard-navigation/

_(Firefox used to have this issue on MacOS too, but I think has since fixed it)_

Reply:
This was actually on Chrome!

Reply:
I see. I think I misunderstood. I thought you were saying you couldn't tab through the nav items. But it sounds like the entire nav menu isn't togglable on mobile w/ keyboard, correct?

If that's the case, that should be solved by the hi-fi design implementation that's currently in progress #313, which includes an overhaul of the nav layout.

Reply:
Basically, when I reduce the width on my computer such that the nav collapses, the three horizontal bars – which toggle the navigation items – is itself not toggle-able. If I click the three bars and open the navigation bar with my mouse,  then I can toggle through the items themselves using the keyboard.

<img width="1018" alt="image" src="https://github.com/user-attachments/assets/48ad8feb-dc84-4309-b1c3-78b3df580f3c" />

Reply:
Yep, I see now. That will be overhauled in the in-progress #313.